Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    exception_notification (4.0.1)
+    exception_notification (4.0.2.ps.1)
```

Comment:
This probably me being pedantic, but I prefer the `4.0.1.ps1` tag notation. It makes it clear that we are branching off 4.0.1 and not some future 4.0.2